_pkgdown.yml

```diff
@@ -1,14 +1,14 @@
-url: https://pharmaverse.github.io/admiraltemplate
+url: https://pharmaverse.github.io/admiralmetabolic
 
-template:
+metabolic:
```

Comment:
i think this was an accidental find/replace - it should be reverted

Reply:
```suggestion
template:
```